test/functional/apps/dashboard/empty_dashboard.js

```diff
@@ -41,10 +41,14 @@ export default function ({ getService, getPageObjects }) {
 
     it('should open add panel when add button is clicked', async () => {
       await testSubjects.click('emptyDashboardAddPanelButton');
+      // have to move the mouse to get the help flyout to appear on IE11
+      // see https://github.com/elastic/kibana/issues/45333
+      console.log('------------------------------------------------------------------ 3');
+      const logo = await testSubjects.find('logo');
+      await logo.moveMouseTo();
       const isAddPanelOpen = await dashboardAddPanel.isAddPanelOpen();
```

Comment:
@LeeDr is it a common issue with `isAddPanelOpen` function?
I wonder if we can put the logic into the function like:
```
async isAddPanelOpen() {
  log.debug('DashboardAddPanel.isAddPanelOpen');
  if (browser.isInternetExplorer) {
    const logo = await testSubjects.find('logo');
    await logo.moveMouseTo();
  }
  return await testSubjects.exists('dashboardAddPanel');
}
```

Reply:
There's an issue mentioned in the comment.  And there's been another similar issue that affects all browsers.  So maybe I can get these other cases fixed and not have to have the moveMouseTo work-around before merging this.  And if I didn't remove that console.log yet, I will.